bothub/project/handle.py

```diff
@@ -0,0 +1,6 @@
+from amqp.channel import Channel
+
+from .consumers.project_consumer import ProjectConsumer
+
+def handle_consumers(channel: Channel) -> None:
+    channel.basic_consume("project.artificial-intelligence", callback=ProjectConsumer().handle)
```

Comment:
By default we name the queue `{resource}.{service}` with `resource` in the plural.
Rename to `"projects.artificial-intelligence"`